File: FoodKing.Services/Database/Menu.cs
using System;
using System.Collections.Generic;

namespace FoodKing.Services.Database;

public partial class Menu
{
    public int Id { get; set; }

    public string Title { get; set; } = null!;
    public DateTime CreatedAt { get; set; } = DateTime.Now;
    public DateTime UpdatedAt { get; set; } = DateTime.Now;
    public bool? SoftDelete { get; set; }

    public virtual ICollection<MenuHasProduct> MenuHasProducts { get; } = new List<MenuHasProduct>();
}

File: UI/foodking_admin/lib/models/menu.dart
class Menu {
  final int? id;
  final String? title;
  final String? createdAt;
  final String? updatedAt;

  Menu({this.id, this.title, this.createdAt, this.updatedAt});

  factory Menu.fromJson(Map<String, dynamic> json) {
    return Menu(
        id: json['id'],
        title: json['title'],
        createdAt: json['createdAt'],
        updatedAt: json['updatedAt']);
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'title': title,
      'createdAt': createdAt,
      'updatedAt': updatedAt
    };
  }
}
